packages/core/ui/ViewContainer.tsx

```diff
@@ -121,84 +125,96 @@ const ViewMenu = observer(
   },
 )
 
-export default observer(
-  ({
-    view,
-    onClose,
-    style,
-    children,
-  }: {
-    view: {
-      menuOptions: MenuOptions[]
-      displayName: string
-      setDisplayName: (displayName: string) => void
-    }
-    onClose: () => void
-    style: React.CSSProperties
-    children: React.ReactNode
-  }) => {
-    const classes = useStyles()
-    const containerNodeRef = useRef<HTMLElement>(null)
+export default withContentRect('bounds')(
+  observer(
+    ({
+      view,
+      onClose,
+      style,
+      children,
+      contentRect,
+      measureRef,
+    }: {
+      view: IBaseViewModel
+      onClose: () => void
+      style: React.CSSProperties
+      children: React.ReactNode
+      contentRect: ContentRect
+      measureRef: React.RefObject<HTMLDivElement>
+    }) => {
+      const classes = useStyles()
+      const theme = useTheme()
+      const padWidth = theme.spacing(1)
```

Comment:
is padWidth required to be equal to the margin used in the theme?

Reply:
Do you mean the classes.viewContainer margin? If so, no, it could be different and this would still work.

Reply:
Ok, I just wasn't wondering if the margin and padding were accidentally being called the same thing interchangeably but if they are different that is ok.